code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -6,77 +6,223 @@
 	item_state = "electronic"
 	flags = CONDUCT
 
+	/// A list of all currently usable and created modules the robot currently has access too.
 	var/list/modules = list()
-	var/obj/item/emag = null
-	var/list/subsystems = list()
+	/// A list of module-specific, non-emag modules the borg will gain when this module is chosen.
+	var/list/basic_modules = list()
+	/// A list of modules ALL robots will get by default, unless the list is overriden.
+	var/list/default_modules = list(/obj/item/flash/cyborg)
```

Comment:
I kinda question the need to keep a separate list just for this, unless there's a good reason for it. Couldn't we just have `/obj/item/flash/cyborg` in each module's `basic_modules` list?

Reply:
That's a good point. I'm going to get rid of it, and stick the flash in `basic_modules`